lib/rouge/lexers/svelte.rb

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Svelte < HTML
+      desc 'Svelte single-file components (https://svelte.dev/)'
+      tag 'svelte'
+      filenames '*.svelte'
+
+      # are these accurate?
+      mimetypes 'text/x-svelte', 'application/x-svelte'
```

Comment:
**suggestion:** Can we add some guessing specs for file names and mime types? 🙏🏼

Reply:
@tancnle I don't know of any filenames outside of `.svelte`, can you elaborate what you're looking for here? (I did remove the comment, that was meant to be removed before :man_facepalming:)

Reply:
Sorry I could have made it a bit clearer. I have added a [commit](https://github.com/rouge-ruby/rouge/pull/1979/commits/44d2237b9ab46c8bb8c7a4e5fc35552a18def82a) to adds the tests. Please let me know what you think.

Reply:
@tancnle whoops, missed the email for your last comment.

Yea, that commit looks good - do you need anything else from me on this?

Reply:
All good. I have added a small commit to add Svelte to the list of support languages https://github.com/rouge-ruby/rouge/pull/1979/commits/d43cab5fa8762a510f944ee3e6cdbfbd0b4e7a76.